agent/build_config.py

```diff
@@ -0,0 +1,174 @@
+import sys
```

Comment:
I don't understand 1. why build_config.py is a file 2.  why it is under the agent folder

Reply:
1. kaggle cli and baseline cli share this, so I move it to a single file for now.

2. Try to put it into utils/config, but meet the recurrent importing issues.

Conclusion 

After I rebuild the kaggle workflow,we need your help to improve the structure (maybe after your reproduce)

Reply:
I will re-structure and refactor the whole project, no worries